docs/source/openshift/exposing-services.adoc

```diff
@@ -0,0 +1,63 @@
+[[exposing-services]]
+= Exposing services
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[overview]]
+== Overview
+
+There are several ways you can expose your service once deployed on OpenShift.
+The following sections describes the various alternatives and when to use them.
+
+[[application-routes]]
+== Routes
+
+Provided you are deploying some form of web application, the most common way to expose it is by a so called link:https://docs.openshift.org/latest/dev_guide/routes.html[route].
```

Comment:
It might not be a web application per se. It might be a micro service which falls also into this category. Which part does bother you here? The "some form of"?